### DISCUSSION TITLE
How can you tell if you have a reproduction game cartridge using this device?

### DISCUSSION THREAD

**Discussion by Cloudy569** [sanni/cartreader#918] [Need help]
What are some of the signs that you have a repro cart?

**Comment by PsychoFox11** [comment]
A reproduction ROM can match a real one 100% - identifying a repro is going to be more in examining the label, PCB, chips used etc.
Of course if it doesn't match a known version that's a sign it may have been tampered with, or it could be an unrecorded revision that needs more research and another copy to verify. But if you got chips from a brand never used or an obvious unofficial PCB, that's your best signs. From a dump alone you aren't gona know if it is a repro or revision though.

Typically even if they had to use a 16MB chip to put a 2MB game on it, the header will say the game is 2MB and it will still dump at that size and maybe they flashed the known ROM to it. But you open it up and look up the chip datasheet and see it is 16MB, they wouldn't have likely wasted all that chip back in the day, but it is more common now on some platforms where smaller chips are harder to find.
